.gitignore

```diff
@@ -3,3 +3,8 @@ coverage.out
 bin/**
 *.log
 .env
+# local build output via go build
+process-compose
```

Comment:
Doesn't the output go to the bin directory?

Reply:
No, it gets symlinked to ./result. I just tried out some of the build commands manually and figured it's safer to prevent checking in the binary accidentally. Absolutely can be removed.